File: GithubJavaPop/app/src/main/java/br/com/githubjavapop/service/PullRequestService.java
package br.com.githubjavapop.service;

import java.util.List;

import br.com.githubjavapop.entidade.PullRequest;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface PullRequestService {
    @GET("/repos/{login}/{name}/pulls")
    Call<List<PullRequest>> getPullRequest(@Path("login") String login, @Path("name") String name);
}

File: app/src/main/java/wilquer/lima/desafioconcrete/network/service/PullRequestService.kt
package wilquer.lima.desafioconcrete.network.service

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path
import wilquer.lima.desafioconcrete.data.model.PullRequest

interface PullRequestService {

    @GET("repos/{creator}/{repo}/pulls")
    fun getPullRequests(@Path("creator") creator: String, @Path("repo") repo: String): Call<List<PullRequest>>
}